### ISSUE TITLE
Beacon region monitoring doesn't work on iPhone 4S on iOS 8

### ISSUE THREAD

**Issue by rszastok** [Estimote/iOS-SDK#154]
I am using iPhone 4S and 3 estimote beacons to test notification demo. Application detect beacons in range, but if I select notification demo -> select beacon  and follow instruction nothing happens. None of "Enter region notification" and "Exit region notification" have ever appeared on my lock screen.

I noticed that:
Method -beaconManager:didEnterRegion: and -beaconManager:didExitRegion: is never called 
Method -switchValueChanged that suppose should be IBOutlet for enterRegionSwitch exitRegionSwitch are not connected. It means that turning ON and OFF enter and exit region does not work.

**Comment by pvormittag** [comment]
I tested this today on an iPhone 4S running iOS 8.1.2 and it did not work. I was not able to receive the expected notifications in either the examples app (which I compiled and ran) or the Estimate app from the AppStore.

I decided to try upgrading to iOS 8.1.3. It appears the issue is solved in iOS 8.1.3 since after upgrading I was able to receive the expected notifications for both the examples app and the Estimote app.
